internal/resources/ekscluster/resource_ekscluster.go

```diff
@@ -550,8 +559,23 @@ func handleClusterDiff(config authctx.TanzuContext, tmcCluster *eksmodel.VmwareT
 		tmcCluster.Meta.Labels = meta.Labels
 	}
 
-	if !reflect.DeepEqual(clusterSpec, tmcCluster.Spec) {
-		updateCluster = true
+	// Check for equality of spec- ignore differences relating to tags generated by TMC
+	if diff := deep.Equal(clusterSpec, tmcCluster.Spec); len(diff) != 0 {
```

Comment:
Why would these tags change in the first place?

Reply:
This was logic I added to compensate for us adding the TMC tags that aren't in the original terraform spec.

Reply:
can we use the suppress diff function which is being used in other places ?

Reply:
Removed